validation-test/SIL/verify_all_overlays.sil

```diff
@@ -1,4 +1,4 @@
-// RUN: for x in %platform-sdk-overlay-dir/*.swiftmodule; do [[ $(basename "$x") = Swift.swiftmodule ]] && continue; llvm-bcanalyzer $x | FileCheck %s; %target-sil-opt -enable-sil-verify-all $x > /dev/null; done
+// RUN: for x in %platform-sdk-overlay-dir/*.swiftmodule; do [[ $(basename "$x") = Swift.swiftmodule ]] && continue; llvm-bcanalyzer $x | FileCheck %s; %target-sil-opt -sdk %sdk -enable-sil-verify-all $x > /dev/null; done
```

Comment:
For more discussion on this change, see: https://github.com/apple/swift/pull/1714/files#r59641994